### PR TITLE
Fix device activation not setting the device class for LoRaWAN 1.0

### PR DESCRIPTION
**Pull request by sp193**
Fix device activation not setting the device class for LoRaWAN 1.0, which causes Class C devices to be always treated as Class A. This happens if the device was activated without going through the join procedure, whereby join.js sets the device mode for LoRaWAN 1.0 devices. This is an attempt for fixing #291.

I will try to get the system tested with a live device tomorrow, but it looks like the device.enabled_class (inside PostgreSQL) is getting updated during device activation. I provide this PR early, so that it may be reviewed/tested independently.

Once again, I do apologize if the code quality is not up to standard, as I major in C and Java.

### CODE REVIEW

File: chirpstack/src/api/device.rs [hunk 1]
```diff
@@ -538,6 +538,13 @@ impl DeviceService for Device {
                 .map_err(|e| e.status())?;
         }
 
+        // LoRaWAN 1.1 devices send a mac-command when changing to Class-C. Change the class here for LoRaWAN 1.0 devices.
+        if dp.supports_class_c && dp.mac_version.to_string().starts_with("1.0") {
+            let _ = device::set_enabled_class(&dev_eui, DeviceClass::C).await;
```
Comment (brocaar) [inline chirpstack/src/api/device.rs:543]: Could you add trailing `.map_err(|e| e.status())?` to both `set_enabled_class` calls to make sure that errors are returned?

Comment (sp193) [reply chirpstack/src/api/device.rs]: Done.

**Comment by sp193** [comment]
I noted that there was a failure of one CI-CD check, which seemed to be a formatting check. So I attempted to fix it according to the diff output.

**Comment by brocaar** [comment]
>  I have correcting the formatting problem, after figuring out that I can trigger the format checker locally with cargo and/or as part of the tests via Make.

Most Rust extensions (e.g. for VSCode) auto-format on save. It saves you a lot of hassle :)

**Pull request merged**